--- configure.orig	2018-04-21 22:21:59.092861289 +0200
+++ configure	2018-04-21 22:22:38.545496200 +0200
@@ -605,7 +605,7 @@
 if enabled_or_auto vaapi; then
   if enabled libav; then
     if check_pkg libva ">=0.38.0"; then
-      check_pkg libva-x11 ">=0.38.0" || die "libva-x11 not found"
+#      check_pkg libva-x11 ">=0.38.0" || die "libva-x11 not found"
       check_pkg libva-drm ">=0.38.0" || die "libva-drm not found"
       enable vaapi
       enable hwaccels
