--- Makefile.orig	2019-01-26 21:28:53.045356233 +0100
+++ Makefile	2019-01-26 21:33:57.713995505 +0100
@@ -16,7 +16,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
-prefix = /usr
+prefix = %install_prefix%
 
 CC=gcc
 
@@ -119,7 +119,6 @@
 distclean:
 	if [ -f tree.o ]; then rm *.o; fi
 	rm -f *~
-	
 
 dist:	distclean
 	tar zcf ../tree-$(VERSION).tgz -C .. `cat .tarball`
